src/storage/db_connection.py

```diff
@@ -34,22 +34,22 @@ def create_tables(self):  # pylint: disable=no-self-use
 
 
 class ReadOnlyConnection(DbConnection):
-    def __init__(self, user: str = 'postgres_ro_user', password: str = 'postgres_ro_pw', **kwargs):
+    def __init__(self, user: str = 'ro_user', password: str = 'ro_pw', **kwargs):
```

Comment:
The weird thing is that the parameters usee and password are not the actual username and password but rather the name of the config key where these can be found.

One could make the argument be the actual username/password, then remove  the argument to all subclasses (since they are unused anyways) and use the global config to load them.

But this is orthogonal to this PR.

Reply:
> One could make the argument be the actual username/password

Yes, that was what I was implying. But yes, this is a bit out of scope for this PR so maybe just a "FixMe"?

Reply:
I opened https://github.com/fkie-cad/FACT_core/issues/1035 to track this